lib/HomeyCompose.js

```diff
@@ -340,6 +341,8 @@ class HomeyCompose {
         if (zwaveParameterIndex) {
           obj[key] = obj[key].replace(/{{zwaveParameterIndex}}/g, zwaveParameterIndex);
         }
+      } else {
```

Comment:
the check if `typeof obj[key]` is an object is now gone

Reply:
yea thats fine since `replaceSpecialPropertiesRecursive` checks it before doing anything.
the only consequence is that we copy over any properties that aren't an object or a string but that seems fine to me.